earthfile2llb/interpreter.go

```diff
@@ -1587,6 +1589,27 @@ func (i *Interpreter) handleImport(ctx context.Context, cmd spec.Command) error
 	return nil
 }
 
+func (i *Interpreter) handleProject(ctx context.Context, cmd spec.Command) error {
+	projectVal, err := i.expandArgs(ctx, cmd.Args[0], false, false)
+	if err != nil {
+		return i.wrapError(err, cmd.SourceLocation, "failed to expand PROJECT %s", cmd.Args[0])
+	}
+
+	if !strings.Contains(projectVal, "/") {
+		return errors.New("format for PROJECT statement should be: <organization>/<project>")
+	}
+
+	parts := strings.Split(projectVal, "/")
+	if len(parts) != 2 {
+		return errors.New("unexpected format for PROJECT statement")
+	}
+
+	i.converter.org = parts[0]
+	i.converter.project = parts[1]
```

Comment:
We try not to use private members of the converter from here - mainly to help create separation. I think we should just have a `Project` method in the converter that sets these. (We typically name the methods in converter by the name of the commands they implement)